src/scripts/main.js

```diff
@@ -1,29 +1,58 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const validTypes = ['success', 'error', 'warning'];
+
+  if (!validTypes.includes(type)) {
+    throw new Error(
+      `Invalid notification type: "${type}". Expected one of ${validTypes.join(', ')}.`,
+    );
+  }
+
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const titles = document.createElement('h2');
+
+  titles.className = 'title';
+  titles.textContent = title;
+  div.appendChild(titles);
```

Comment:
The variable name 'titles' is misleading as it suggests that it might contain multiple titles, while it contains only one. Consider renaming it to 'titleElement' or just 'title'.